packages/database/src/core/util/util.ts

```diff
@@ -566,6 +566,42 @@ export const tryParseInt = function (str: string): number | null {
   return null;
 };
 
+const MIN_PRINTABLE_CHAR: number = 0x20;
```

Comment:
This should be changed to only allow the blessed characters: https://stackoverflow.com/questions/36086317/firebase-push-id-what-characters-can-be-inside